### DISCUSSION TITLE
Size 3D Scene to fit parent container

### DISCUSSION THREAD

**Discussion by AndrewMHenry** [zauberzeug/nicegui#1163] [Ideas / Feature Requests]
### Question

Hello,

Is there a way to create an instance of the`ui.scene()` element (a very cool element, BTW 😁) that automatically sizes to fit the parent container?

**Comment by falkoschindler** [comment]
Hi @AndrewMHenry!
Currently there is no way to do that. But since we started implementing `ui.scene` ages ago, I wished we could adjust its size dynamically to fit the container. It's not trivial, because there's a canvas that needs to know its size up front. But we could probably detect when the container div resizes and resize the canvas accordingly.
Let's convert this discussion to a feature request.

**Reply by AndrewMHenry** [reply]
Hi @falkoschindler, thanks for clarifying. I agree that would be a great feature!

**Comment by falkoschindler** [comment]
I found a solution in PR https://github.com/zauberzeug/nicegui/pull/1190 which will be release shortly. 🥳

**Reply by AndrewMHenry** [reply]
Awesome, thanks for the speedy response, @falkoschindler ! 🥳
